### PR TITLE
Feature/img placeholder

### PR DESCRIPTION
**Pull request by Mariusz-web**

### CODE REVIEW

File: src/components/ImageComponent/ImageComponent.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
 import { observerListener } from '../../shared/observer';
 
 const template = (imageUrl, className, id) => {
-  return `
-    <img src="${imageUrl}" class="${className}" id="${id}"/>
-  `
+  let src = true;
```
Comment (Veryyapeee) [inline src/components/ImageComponent/ImageComponent.js]: 1. If(src)? 
You just setted up it to true, this condition would always be true
2. You forget to wrap up the statement in return with ``
3. You can go just with in src in img

imgUrl || 'link to placeholder'

**Review by Veryyapeee — CHANGES_REQUESTED** [review]

File: src/components/ImageComponent/ImageComponent.js (hunk 1 above)
Comment (Mariusz-web) [reply src/components/ImageComponent/ImageComponent.js]: changed it

**Review by maciejopalinski — APPROVED** [review]

**Comment by Veryyapeee** [comment]
Something is not working, did you even checked that?

Debug the response from API and check how the data looks like when we don't have an image

**Comment by maciejopalinski** [comment]
> 
> 
> Something is not working, did you even checked that?
> 
> Debug the response from API and check how the data looks like when we don't have an image

@Veryyapeee that is just an example of incorrect data from API. Image URL is just incorrect, but it EXISTS.
Look:
image

Is there a way of knowing if image tag did not render due to 404 error?

**Comment by Veryyapeee** [comment]
@PoProstuMieciek 
Image has onLoad method, and complete property, you can try to work with that

And onerror I think, if I remember correctly, I'm driving to the vet with cat right now and can't check xd

**Review by Veryyapeee — APPROVED** [review]

**Pull request merged**